katz/src/main/kotlin/katz/effects/Duration.kt

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright [2016] [sksamuel]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package katz
+
+import java.util.concurrent.TimeUnit
+
+data class Duration(val amount: Long, val timeUnit: TimeUnit) {
+    val nanoseconds: Long by lazy { timeUnit.toNanos(amount) }
+
+    companion object {
+        // Actually limited to 9223372036854775807 days, so unless you are very patient, it is unlimited ;-)
+        @JvmStatic val INFINITE = Duration(amount = Long.MAX_VALUE, timeUnit = TimeUnit.DAYS)
+    }
+}
+
+val Long.days: Duration
```

Comment:
Ok heheheh. Didn't see it.
![image](https://user-images.githubusercontent.com/6547526/27041187-83b740f4-4f93-11e7-8b25-11794efc0e07.png)